### PR TITLE
DONOT MERGE- just review Newforecasts

### PR DESCRIPTION
**Pull request by MillsProvosty**
## Changes proposed in this pull request:

Resolves #

## The following checks have been completed:

 - [ ] Tested new feature(s) as well as any feasible edge cases
 - [ ] Ran the test suite - all tests are passing
 - [ ] Checked affected endpoints in Postman

### CODE REVIEW

**Comment by MillsProvosty** [comment]
@smainar - I've deleted most of what went in here, however, if you have any tips or helpful tutorials, that you could add to the review, that'd be great.

File: app.js [hunk 1]
```diff
@@ -2,11 +2,13 @@ var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
+require('dotenv').config()
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/api/v1/users');
 var sessionsRouter = require('./routes/api/v1/sessions');
-
+var forecstsRouter = require('./routes/api/v1/forecasts');
```
Comment (smainar) [inline app.js]: `var forecstsRouter` should be `var forecastsRouter` -- it is missing the letter 'a'

File: app.spec.js [hunk 2]
```diff
@@ -14,37 +14,69 @@ describe('api', () => {
...
-      return request(app).post("/api/v1/users")
-      .send({email: "email@email.com", password: "password"})
-      .expect(function(res) {
-        expect(response.statusCode).toBe(201)
-      })
-    });
-  });
-});
+/* POST forecasts */
+describe('Tests the forecasts Post path', () => {
```
Comment (smainar) [inline app.spec.js]: This is a `GET` path

File: routes/api/v1/forecasts.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+//
+// }).on("error", (err) => {
+//   console.log("Error: " + err.message);
+// });
+/* POST returns forecast */
+
+router.post("/", function(req, res, next){
+  res.setHeader("Content-Type", "application/json")
+  if (req.body.api_key !== undefined){
+    user.findOne({
```
Comment (smainar) [inline routes/api/v1/forecasts.js]: `user` should be `User`, otherwise, the `.findOne()` method cannot attempt to find the particular record in your database that matches the given criteria

File: routes/api/v1/forecasts.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      where: {
+        apiKey: req.body.api_key
```
Comment (smainar) [inline routes/api/v1/forecasts.js]: Is this currently working? If not, I think it's suppose to be:

```
api_key: req.body.apiKey
```
